lib/msal-browser/src/cache/BrowserCacheManager.ts

```diff
@@ -1696,10 +1690,10 @@ export class BrowserCacheManager extends CacheManager {
             throw BrowserAuthError.createNoTokenRequestCacheError();
         }
 
-        const parsedRequest = this.validateAndParseJson(
-            browserCrypto.base64Decode(encodedTokenRequest)
-        ) as CommonAuthorizationCodeRequest;
-        if (!parsedRequest) {
+        let parsedRequest: CommonAuthorizationCodeRequest;
+        try {
+            parsedRequest = JSON.parse(base64Decode(encodedTokenRequest));
+        } catch (e) {
```

Comment:
Include the caught exception inside the rethrow?

Reply:
Added a couple log messages instead.